src/assets/samples/courses.json

```diff
@@ -736,9 +736,6 @@
           {
             "subtitle": "Deep Learning"
           },
-          {
-            "subtitle": "NLP"
-          },
```

Comment:
Was this intentionally removed?

Reply:
yes we will not do nlp in the cologn course missed this the first time